buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -3,6 +3,43 @@
     {
       "enabled": true,
       "continueOnError": false,
+      "alwaysRun": true,
```

Comment:
The reason to do cleanup first is to unblock the current agent to do the build without running out of space. We don't have much data about why these errors are happening, so I don't think we can say for sure if doing cleanup after the build will help anything. Also, I believe docker cleanup (at least) is done before the build for CoreFX and CoreCLR. (For CoreFX, as part of init-docker.)